WooCommerce/Classes/ViewRelated/Blaze/BlazeIntro/BlazeLearnHowView.swift

```diff
@@ -0,0 +1,242 @@
+///
+struct BlazeLearnHowView: View {
+    @Binding var isPresented: Bool
+
+    private let steps: [Step] = [.init(number: Localization.ChooseProduct.number,
+                                       title: Localization.ChooseProduct.title,
+                                       subtile: Localization.ChooseProduct.subtitle),
+                                 .init(number: Localization.CustomizeTargeting.number,
+                                       title: Localization.CustomizeTargeting.title,
+                                       subtile: Localization.CustomizeTargeting.subtitle),
+                                 .init(number: Localization.SetYourBudget.number,
+                                       title: Localization.SetYourBudget.title,
+                                       subtile: Localization.SetYourBudget.subtitle),
+                                 .init(number: Localization.QuickReview.number,
+                                       title: Localization.QuickReview.title,
+                                       subtile: Localization.QuickReview.subtitle),
+                                 .init(number: Localization.GoLive.number,
+                                       title: Localization.GoLive.title,
+                                       subtile: Localization.GoLive.subtitle)]
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 0) {
+            titleBlock
+
+            stepBlock
+        }
+    }
+}
+
+private extension BlazeLearnHowView {
+    var titleBlock: some View {
+        ZStack(alignment: .center) {
+            HStack {
+                Button(action: {
+                    isPresented = false
+                }, label: {
+                    Image(uiImage: .closeButton)
+                        .secondaryBodyStyle()
+                })
+
+                Spacer()
+            }
+            .padding(.horizontal, Layout.TitleBlock.closeButtonHorizontalPadding)
+
+            HStack {
+                Text(String(format: Localization.title, "Blaze"))
+                    .fontWeight(.semibold)
+                    .headlineStyle()
+            }
+        }
+        .padding(.top, Layout.TitleBlock.topPadding)
+    }
+
+    var stepBlock: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0) {
+                VStack(alignment: .leading, spacing: 0) {
+                    ForEach(steps, id: \.title) { step in
+                        StepView(step: step)
+
+                        Divider()
+                            .frame(height: Layout.StepBlock.dividerHeight)
+                            .overlay(Color(uiColor: .systemBackground))
+                    }
+                }
+                .background(Layout.StepBlock.backgroundColor)
+            }
+            .cornerRadius(Layout.StepBlock.cornerRadius)
+        }
+        .padding(Layout.contentPadding)
+    }
+}
+
+private extension BlazeLearnHowView {
+    struct Step {
+        let number: String
```

Comment:
Should number be an Int instead of String?

Reply:
Updated in https://github.com/woocommerce/woocommerce-ios/pull/11583/commits/5697ce1a49863666dbdac981540c3541b9e3b1ae